addons/project_issue/security/ir.model.access.csv

```diff
@@ -4,6 +4,6 @@ access_project_issue_report_all,project.issue.report,model_project_issue_report,
 access_resource_calendar_project_manager,resource.calendar.project.manager,resource.model_resource_calendar,project.group_project_manager,1,1,1,1
 access_project_issue_report_user,project.issue.report user,model_project_issue_report,project.group_project_user,1,0,0,0
 access_crm_team,crm.team,sales_team.model_crm_team,project.group_project_user,1,0,0,0
-access_project_issue_salesman,project.issue,model_project_issue,base.group_sale_salesman,1,0,0,0
-access_project_issue_manager,project.issue,model_project_issue,base.group_sale_manager,1,0,0,0
+access_project_issue_salesman,project.issue,model_project_issue,sales_team.group_sale_salesman,1,0,0,0

```

Comment:
By the way: why do we have a dependency between sales team and project issue ?

Looking at the code

 - team_id field is never used
 - team_id parameter of stage_find is actually a project_id, because stages depends on the project, not sales team